cmd/utils/flags.go

```diff
@@ -447,6 +451,33 @@ func MakeEthConfig(clientID, version string, ctx *cli.Context) *eth.Config {
 		SolcPath:                ctx.GlobalString(SolcPathFlag.Name),
 		AutoDAG:                 ctx.GlobalBool(AutoDAGFlag.Name) || ctx.GlobalBool(MiningEnabledFlag.Name),
 	}
+
+	if ctx.GlobalBool(DevModeFlag.Name) {
+		if !ctx.IsSet(VMDebugFlag.Name) {
```

Comment:
Any reason why you are using `ctx.IsSet` instead of `ctx.GlobalIsSet` since these are global flags?